lib/src/doc/lives.rs

```diff
@@ -100,42 +96,160 @@ impl<'a> Document<'a> {
 				// Finally, let's check what type of statement
 				// caused this LIVE query to run, and send the
 				// relevant notification based on the statement.
+				let mut tx = txn.lock().await;
+				let ts = tx.clock().await;
+				let ns = Box::new(opt.ns().to_string());
+				let db = Box::new(opt.db().to_string());
+				let tb = Box::new(self.id.unwrap().tb.to_string());
+				let not_id = crate::sql::Uuid::new_v4();
 				if stm.is_delete() {
 					// Send a DELETE notification
+					let thing = (*rid).clone();
+					let notification = Notification {
+						live_id: lv.id.clone(),
```

Comment:
UUIDs are `Copy`, no need to clone here.

Reply:
Turns out it wasn't `Copy`, so I added it. But then it turned out dereferencing changed the type from `sql::Uuid` to `uuid::Uuid`. Fixing that though, thanks for spotting!